packages/core/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex, Box } from 'pcln-design-system'
+
+const CustomBox = styled(Box)`
+  border-radius: 2px;
+`
+
+const defaultStepColor = 'background.light'
+
+function ProgressBar({ steps, currentStep, stepHeight }) {
+  return (
+    <Flex>
```

Comment:
can you add a className prop to this flex component please this'll allow us to use styled components on the progress bar should we need to